more_itertools/more.py

```diff
@@ -1248,7 +1247,7 @@ def numeric_range(*args):
         >>> list(numeric_range(start, stop, step))
         [Fraction(1, 2), Fraction(1, 1), Fraction(3, 2), Fraction(2, 1)]
 
-    Negative steps are supported:
+    If *step* is zero, ``ValueError`` is rasied. Negative steps are supported:
```

Comment:
And the second sentence could start with `But negative ...` to emphasis the relation to zero-steps.